tests/models/decoder_only/vision_language/test_models.py

```diff
@@ -327,6 +327,21 @@
         vllm_output_post_proc=model_utils.qwen_vllm_to_hf_output,
         prompt_path_encoder=model_utils.qwen_prompt_path_encoder,
     ),
+    "idefics3": VLMTestInfo(
+        models=["HuggingFaceM4/Idefics3-8B-Llama3"],
+        test_type=(VLMTestType.IMAGE, VLMTestType.MULTI_IMAGE),
```

Comment:
Since Idefics3 also supports `MULTI_IMAGE` inputs, can you update the `examples/offline_inference_vision_language_multi_image.py` as well?